code/game/turfs/simulated/floor/reinf_floor.dm

```diff
@@ -34,6 +34,9 @@
 		ChangeTurf(/turf/open/floor/plating)
 	return TRUE
 
+/turf/open/floor/engine/attackby(obj/item/C, mob/user, params)
```

Comment:
This is not a proper fix, move force param in make_plating up to /floor and pass 1 in manual removal.

Reply:
@AnturK 

But we don't want people to be able to crowbar reinforced flooring either.

Reply:
return on crowbar_act then?

Reply:
Oh, floor doesn't use crowbar_act and calls prytile directly if you attack it with a crowbar, hm.